File: cp1.py
n = int(input("Enter your Number"))
b = []
b.append(n)
while(n != 1):
    if(n%2 == 0):
        n = n/2
        b.append(n)
    else:
        n = n*3 + 1
        b.append(n)

print(b)
# shubham jadhao

File: CP1.java
import java.util.Scanner;
public class CP1{
	public static void main(String[] args) {
		Scanner in= new Scanner(System.in);
		System.out.println("Enter number");
		int n=in.nextInt();
		System.out.printf("%d ",n);
		while(n!=1){
			if (n%2==0) {
				n=n/2;
				System.out.printf("%d ",n);
			}
			else{
				n=n*3+1;
				System.out.printf("%d ",n);
			}
		}
	}
}